gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/ConditionMapperTest.java

```diff
@@ -396,6 +396,32 @@ void When_MappingLinksetWhichIsAReferralRequestToExternalDocumentLinkSet_Expect_
         assertThat(conditions).isEmpty();
     }
 
+    @Test
+    void When_MappingLinksetWithNotMajorCode_Expect_DefaultedToMinor() {
+        final var ehrExtract = unmarshallEhrExtract(
+                "Condition",
+                "linkset_with_minor_severity_code.xml"
+        );
+
+        final List<Condition> conditions = conditionMapper.mapResources(ehrExtract, patient, List.of(), PRACTISE_CODE);
+
+        assertThat(conditions).isNotEmpty();
+        assertThat(conditions.getFirst().getNote()).hasSize(1);
+        assertThat(conditions.getFirst().getNote().getFirst().getText()).isEqualTo("Defaulted to Minor");
```

Comment:
I think assertEquals will be shorter and more readable here. Also, "Defaulted to Minor" can be made a constant